src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java

```diff
@@ -144,6 +144,11 @@ private static void logEvent(String message, String userEmail, String type) {
                 accessKey = PLUGIN_USER_ACCESS_KEY;
                 secretKey = PLUGIN_USER_SECRET_KEY;
             }
+
+            if (accessKey == null || secretKey == null) {
+                System.out.println("Plugin is not able communicate with the server. Please check your internet connection.");
```

Comment:
Internet connection error?

Reply:
Yes, If user is not connected to the internet then these will be `null`. I added this so that plugin does not error our and keeps tracking changes until user is back online and sync resumes.